WordPress/wordcamp.org
# Add banner to previous year events redirecting visitors to the latest event

Add notification if a user visits WordCamp site that has a newer site for the same location. Keeps the canonical link functionality for seo purposes.

Also renamed the file to follow better naming, as it made sense to put this functionality into the same file that already contained hints about the latest site for search engines.

This was a bit tricky to test because of the differences between the Docker environment and production. Also, the development database is lacking proper real-world examples. So needs proper thorough testing, mentioning @iandunn here as they had written the seo code on which this PR is based.

Fixes #688

Props @dd32 

### Screenshots

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/415544/148633354-6b83dfc9-9cd8-4913-8059-b861ef765bdc.png">

### How to test the changes in this Pull Request:

1. Create new WordCamp entry on tracker, based on some other existing event but change the year
2. Make sure the new WordCamp has a site
3. Visit the WordCamp site from which you based the new event, notification should be visible
